README.rst

```diff
@@ -165,15 +104,15 @@ report bugs, and request new features. This is the best place to post questions
 Distribution
 ------------
 
-If you want to create either source or wheel files, start by installing
-the building requirements and then executing the build module:
+If you want to create either source or wheel files, start by installing poetry and then 
+executing the build module. To install poetry we suggest using the pipx installation
```

Comment:
```suggestion
executing the build module. To install poetry, you should use the pipx installation
```
Google developer documentation style guide (and most tech writing style guides) say avoid ``we``.